components/forms/cct/CalcForm.tsx

```diff
@@ -23,7 +23,7 @@ export type CalcFormValues = {
   ftePercents: FtePercentsTypes[];
   propStartDate: string;
   propEndDate: string;
-  currentFtePercent: string;
+  currentFtePercent: number | string;
```

Comment:
just had to slip this tweak in, eh? 😂

Reply:
All killer no filler, my friend.
There was an error in the `AutocompleteSelect handleChange` function (incorrectly targeting `val.label` instead of the `val.value`) which then developed into a game of splat the rat. The rat may have won.

Reply:
🐀 🏅 😂